src/buildCommon.js

```diff
@@ -47,8 +47,12 @@ var makeSymbol = function(value, fontFamily, mode, options, classes) {
 
     var symbolNode;
     if (metrics) {
+        var italic = metrics.italic;
+        if (mode === "text") {
+            italic = 0;
+        }
```

Comment:
I don't think it will. In the MathDefaultFonts test (that image), italic correction SHOULD be applied for every character.

BTW I have some explanations for questions in #349. KaTeX's \mathit command incorrectly selects the math italic font. The LaTeX \mathit command actually selects the TEXT italic font, which has somewhat narrower letters than true math italic. \mathit means “use text italic in math mode”, it doesn't mean “use math italic.“ Similarly for \mathrm, \mathtt, etc. From fontmath.ltx:

```
\DeclareMathAlphabet      {\mathbf}{OT1}{cmr}{bx}{n}
\DeclareMathAlphabet      {\mathsf}{OT1}{cmss}{m}{n}
\DeclareMathAlphabet      {\mathit}{OT1}{cmr}{m}{it}
\DeclareMathAlphabet      {\mathtt}{OT1}{cmtt}{m}{n}
```